test/server/__snapshots__/Server.test.js.snap.webpack5

```diff
@@ -45,8 +43,7 @@ Array [
   Array [
     "client",
     "index.js?http:",
-    "0.0.0.0&path=",
-    "ws&port=8100",
+    "0.0.0.0",
   ],
   Array [
```

Comment:
Existing snapshots should not be changed, it is big breaking change

Reply:
oh yep need normalize